docs/usage/special_features.rst

```diff
@@ -0,0 +1,42 @@
+Special Features
+#################
+
+Re:dash has a lot of very useful features and most of them can be found easily when using the UI. This page features the less well-known ones.
+
+Queries
+========
+It is possible to have filters for query results and visualizations. Thanks to filters, you can restrain the result to a certain or multiple values. Filters are enabled by following a naming convention for columns.
+
+If you want to focus only on a specific value, you will need to alias your column to ``<columnName>::filter``. Here is an example:
+
+.. code:: sql
+
+    select action as "action::filter", count (0) as "actions count"
+    from events
+    group by action
+
+You can see this query and the rendered UI `here <http://demo.redash.io/queries/143/source#table>`_.
+
+If you are interested in multi filters (meaning that you can select multiple values), you will need to alias your column to ``<columnName>::multi-filter``. Here is an example:
+
+.. code:: sql
+
+    select action as "action::multi-filter", count (0) as "actions count"
+    from events
+    group by action
+
+You can see this query and the rendered UI `here <http://demo.redash.io/queries/144/source#table>`_.
+
```

Comment:
Worth mentioning that we support `__filter` and `__multiFilter` for databases that don't support `::` in column names (like BigQuery).